app/config/config.test.json

```diff
@@ -40,7 +40,8 @@
 		"project-id": "myprojectid",
 		"background-color": "000000",
 		"skin": "default",
-		"theme": "default"
+		"theme": "default",
+		"testmode": true
```

Comment:
This should also be added to `config.dist.json` which is meant as a kind of "template" for the real production files. if It's missing, there will be fatal type errors in `Presentation/CreditCardConfig.php`